contract/test/BankBeacon.spec.ts

```diff
@@ -17,7 +17,23 @@ describe('BankBeacon', () => {
     await bankImplementation.waitForDeployment()
 
     const encodedInitialize = bankImplementation.interface.encodeFunctionData('initialize', [
-      await tipsProxy.getAddress()
+      await tipsProxy.getAddress(),
+      await superAdmin.getAddress()
+    ])
+
+    const encodedInitialize1 = bankImplementation.interface.encodeFunctionData('initialize', [
+      await tipsProxy.getAddress(),
+      await user1.getAddress()
+    ])
+
+    const encodedInitialize2 = bankImplementation.interface.encodeFunctionData('initialize', [
+      await tipsProxy.getAddress(),
+      await user2.getAddress()
+    ])
+
+    const encodedInitialize3 = bankImplementation.interface.encodeFunctionData('initialize', [
+      await tipsProxy.getAddress(),
+      await user3.getAddress()
```

Comment:
Why not getting tips address as a constant and use it for next line. It will be simple and easy to read

Reply:
And name the encodeInitialize in a way you know what you are doing. 

Reading encodeInitialize3 dont explain what the difference between 2 and 3

Reply:
Applied your suggestions, thank you!